students/guillaume/quarter_2_project/leankitbackend/leankitbackend/dataprocess.py

```diff
@@ -0,0 +1,311 @@
+#!/usr/bin/env python3
+from leankitbackend.leankitcards import (cards_per_lanes, workload_users,
+                                         cards_per_users, projects,
+                                         cards_per_projects, project_size)
+from leankitbackend.leankitcards import project_in_work
+from prettytable import PrettyTable
+from leankitbackend.date_analysis import delta_date, cvt_d
+
+'''
+http://zetcode.com/python/prettytable/
+'''
+
+
+lanes_current_sprint = ['Current Sprint Backlog', 'Doing Now', 'Under Review',
+                        'Finished As Planned']
+
+
+def assigned(card, attr='assignedUsers'):
+    name_str = ', '.join([assigned['fullName']
+                          for assigned in getattr(card, attr)])
+    return name_str
+
+
+def created_by(card):
```

Comment:
why the wrapper, rather than having a method or property on the card class that does this?